resources/elbv2_load_balancers.go

```diff
@@ -110,7 +115,6 @@ func Elbv2LoadBalancers() *schema.Table {
 				Name:        "aws_elbv2_load_balancer_availability_zones",
 				Description: "Information about an Availability Zone.",
 				Resolver:    fetchElbv2LoadBalancerAvailabilityZones,
-				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"load_balancer_cq_id", "zone_name"}},
```

Comment:
Why did you remove the PK? if it didn't cause an issue I see no reason to remove for no reason.